backend/services/frontend/handler/frontend.go

```diff
@@ -160,11 +183,15 @@ func (e *Frontend) CreateGroupHandler(c *fiber.Ctx) error {
 
 func (e *Frontend) ModifyGroupHandler(c *fiber.Ctx) error {
 	var data map[string]*string
+	var groupType pbCollaboration.GroupType
 	if err := c.BodyParser(&data); err != nil {
 		return err
 	}
 	userID := helper.GetUserIDFromContext(c)
-	rspModifyGroup, err := e.collaborationService.ModifyGroup(c.Context(), &pbCollaboration.ModifyGroupRequest{UserID: userID, GroupID: c.Params("groupID"), GroupName: data["groupName"]})
+	if data["groupType"] != nil {
+		groupType = converter.MigrateStringGroupTypeToProtoGroupType(*data["groupType"])
+	}
```

Comment:
tip: you could also do:

```suggestion
	if gt := data["groupType"]; gt != nil {
		groupType = converter.MigrateStringGroupTypeToProtoGroupType(*gt)
	}
```

Reply:
Good idea, applied in 2b260807d3537dee83e2fef7df298c1c6cdd2d30